apex/contrib/csrc/multihead_attn/encdec_multihead_attn_norm_add_cuda.cu

```diff
@@ -374,9 +357,8 @@ std::vector<torch::Tensor> bwd_cuda(
   char b_layout_n{'n'};
   char b_layout_t{'t'}; 
 
-  rocblas_int flags = 0;
-  
   //TORCH_CUDABLAS_CHECK(cublasSetMathMode(handle, CUBLAS_TENSOR_OP_MATH));
+  /*
```

Comment:
Better to delete this code, instead of commenting since it is NOT required, as the flag parameter is no more used in the new hipblas_gemm_ex() API.